src/commands/_shared.ts

```diff
@@ -9,6 +9,13 @@ export const sharedArgs = {
   },
 } as const
 
+export const envNameArgs = {
+  envName: {
+    type: 'string',
+    description: 'Name of the build environment to use (see \'Environment overrides\' in the docs)',
```

Comment:
I would suggest something that is quickly understandable in usage

```suggestion
    description: 'Environment variable to use for resolving config (default is `NODE_ENV`)',
```

Reply:
Agreed, however, the description you suggest does not match the implementation. The value passed to the  envName argument does not match an env variable but the environment name in the Nuxt configuration. This behaviour matches the behaviour of c12.

Reply:
Indeed c12 does config resolution right? it is too much impl detail if we want to mention low level library IMO. Any better ideas is welcome!

Reply:
In that case, I would suggest something like 'Name of the environment overrides to use for resolving config'. This uses the current wording of the Nuxt configuration, here: https://nuxt.com/docs/getting-started/configuration#environment-overrides